code/datums/components/rot.dm

```diff
@@ -115,15 +118,18 @@
 	if(!active)
 		return
 
+	var/time_delta = world.time - start_time
 	// Wait a bit before decaying
-	if(world.time - start_time < start_delay)
+	if(time_delta < start_delay)
 		return
 
-	if(!prob(strength * 3))
+	var/time_scaling = min((time_delta - start_delay) / scaling_delay, 1)
+
+	if(!prob(strength * 1 * time_scaling))
```

Comment:
`* 1`?

Reply:
I'm trying to keep the format of scalar * scalar * base_value here, I can yeet this if you'd like tho

Reply:
Nah, just wanted to make sure it wasn't a mistake.